pkg/api/validation/validation.go

```diff
@@ -30,12 +30,29 @@ import (
 	"github.com/golang/glog"
 )
 
+const MinPort int = 1
+const MinHostPort int = 0
+const MaxPort int = 65536
+
+const intervalValueErrorMsg string = "must be %d or more and less than %d"
+
+const QualifiedNameErrorMsg string = "must match regex [" + util.Dns1123SubdomainFmt + " / ] " + util.Dns1123LabelFmt
```

Comment:
DNSLabel and DNSSubdomain have max lenghts - the error should say what the length is, just like the others.  "must be less than %d characters and match regex %s"